### PR TITLE
[CDAP-8965, 8966, 8970] Minor bugfixes on Home list view, Pipelines, and Market

### PR DESCRIPTION
**Pull request by tonybach**
[CDAP-8965](https://issues.cask.co/browse/CDAP-8965) Remove Wrangle button from Wrangler Transform
[CDAP-8966](https://issues.cask.co/browse/CDAP-8966) Fix layout issues on overview page

#### EDIT: Added few bug fixes.
[CDAP-8970](https://issues.cask.co/browse/CDAP-8970) - Fix links in informational step in cask wizard.
Fixes PluginUpload wizard error title to not show "Third Party Artifact" in title

### CODE REVIEW

**Comment by tonybach** [comment]
Since it's hard to locally test the retry message, here's what it looks like after the fix, which is also what it was like before the bug appeared:

screen shot 2017-03-13 at 6 26 23 pm

File: cdap-ui/app/cdap/components/EntityListView/EntityListInfo/EntityListInfo.scss [hunk 1]
```diff
@@ -42,6 +42,7 @@ $hovered-page-background-color: rgba(219, 219, 219, 1);
       font-size: 12px;
       display: inline-block;
       vertical-align: top;
+      transform: translateY(3px);
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/EntityListView/EntityListInfo/EntityListInfo.scss]: This shouldn't be required. Just remove both `vertical-align` and `transform`? That would just align things right?
screen shot 2017-03-14 at 11 25 13 am

This is what I'm seeing after removing those two.

**Review by ajainarayanan — COMMENTED** [review]
Minor comment.

File: cdap-ui/app/cdap/components/EntityListView/EntityListInfo/EntityListInfo.scss (hunk 1 above)
Comment (ajainarayanan) [reply cdap-ui/app/cdap/components/EntityListView/EntityListInfo/EntityListInfo.scss]: Tested it in FF and the pagination `ul` needs a `vertical-align: top`

File: cdap-ui/app/cdap/components/CaskWizards/MarketHydratorPluginUpload/index.js [hunk 2]
```diff
@@ -61,6 +61,7 @@ export default class MarketHydratorPluginUpload extends Component {
             jsonFile
           }
         });
+        throw 'fail';
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/CaskWizards/MarketHydratorPluginUpload/index.js]: what was this addition for?

Comment (ajainarayanan) [reply cdap-ui/app/cdap/components/CaskWizards/MarketHydratorPluginUpload/index.js]: oh! thats wrong. Used for debugging. Sorry will remove it. Good catch

**Comment by tonybach** [comment]
^ Besides the minor comment, @ajainarayanan's and @elfenheart's changes LGTM 👍

File: cdap-ui/app/cdap/components/EntityListView/EntityListInfo/EntityListInfo.scss (hunk 1 above)
Comment (tonybach) [reply cdap-ui/app/cdap/components/EntityListView/EntityListInfo/EntityListInfo.scss]: Tried several other alternatives, all of which broke either Chrome's or Firefox's alignment

**Comment by elfenheart** [comment]
👍  LGTM

**Comment by leacd** [comment]
LGTM

**Pull request merged**